### ISSUE TITLE
Google sign in stuck after asking app specific permissions

### ISSUE THREAD

**Issue by cdfa** [BranchMetrics/sesame_issues#46]
Hi, I was just trying out the app today, but when trying to add a Google account, I only get as far as the popup where I can select YouTube, calendar etc. When I click "sign in" it shows a loader, but it never ends.

I hope you get to the bottom of this before my trial runs out, but I fear you won't, so I wouldn't be able to try out this functionality.

**Comment by philwall2** [comment]
Yeah it sucks. We're hitting our quota for the YouTube API (silver lining is it's happening because the app is growing).

We're blacked out between 7pm and 12am San Francisco time, then our quota resets. You can workaround by connecting outside that window.

We've asked Google to increase our YouTube quota, no idea if they will.

We're also going to reduce the # of YouTube data updates we're running with existing users.

Sorry about the hassle.

**Comment by cdfa** [comment]
Wow haha, a silver lining indeed. I'll on other hours.
If you don't know if Google is going to increase you're quota, maybe you should consider detecting this and letting the user know.

**Comment by twotrident** [comment]
> You can workaround by connecting outside that window.

Where can I connect outside that window?

**Comment by philwall2** [comment]
If you're having connection problems now, this is a different problem. Google is blocking us because they claim we haven't followed one of their new policies that went into effect this week. We think we have and are appealing. Sorry. TBH, we're pretty tired of dealing with them but have no choice.

**Comment by VK6DVR** [comment]
Sesame settings Calendar Missing Permission
Cannot successfully finish the connection to Google accounts.
Google apps | +ADD ACCOUNT
Add an account to get shortcuts for Calendar and YouTube.
In choosing 1 of 4 installed accounts, the dialog crashes to blank extremely fast, nothing gets chosen.

**Comment by philwall2** [comment]
Sorry you have to deal with this. Google throttles us and stops allowing account connections. It's extremely frustrating. We were granted a larger # of connections, then randomly they took them away and won't explain why.
